src/ai/backend/manager/container_registry/base.py

```diff
@@ -164,8 +164,11 @@ async def commit_rescan_result(self) -> list[ImageRow]:
                         image_row.config_digest = update["config_digest"]
                         image_row.size_bytes = update["size_bytes"]
                         image_row.accelerators = update.get("accels")
+
+                        if image_row.labels != update["labels"]:
+                            image_row.resources = update["resources"]
```

Comment:
1. image rescan -> update labels only, not resources
2. when user creates -> update only resources
3. if no value in resources, check based on labels
	* if resources is specified as property, return value from resources, otherwise parse and return value from labels